server/lib/enrich/article.js

```diff
@@ -31,6 +31,7 @@ module.exports = exports = function article(content, format) {
 		content.bodyHTML = content.body;
 	}
 
+	content.wordCount = content.contentStats && content.contentStats.wordCount;
```

Comment:
You've deleted the use of `getWordCount` home brew function 🎉 , but I can't see you deleting the function itself. I presume it is used in a few other places and we'll need to tidy those instances up before deleting the function?